docs/welcome/js/welcome.js

```diff
@@ -31,7 +31,7 @@
           as they enter and exit the view.
         `
       ],
-      attachTo: '.hero-welcome bottom',
+      attachTo: '.hero-welcome bottom-start',
```

Comment:
@chuckcarpenter I'm seeing a few issues in the demo app:

<img width="854" alt="screen shot 2018-10-10 at 2 16 09 pm" src="https://user-images.githubusercontent.com/5483853/46757095-205e9e80-cc97-11e8-8610-f22f104d8116.png">

Did you mean to add the `-start` here?

Reply:
Also, I think we still need to add calculations for `-start` and `-end`.

Reply:
@chuckcarpenter @rwwagner90 Perhaps it would be best to simply _add_ to our existing Popper arrow styles for now (i.e., adding calculations for the `-start` and `-end` positions). 

We could make fully converting to Tippy arrows a part of 2.X or 3.0.

Reply:
@BrianSipple I'm okay with either, but it might be simpler to fully convert to tippy arrows now, rather than trying to use the old styles. I'll leave it up to you guys 😃

Reply:
@BrianSipple I think this is easier than adding the additional calculations though? This is a setting and falls back to Tippy options.

Reply:
Adding the additional calculations would mean we don't risk breaking things like the positioning of the main content and aesthetic appearance of the arrows. Ultimately, using Tippy directly would be cleaner, but I feel like we'd be better off focusing on that after shipping 2.0.

Reply:
@chuckcarpenter @rwwagner90 Setting the position to `bottom-start` still produces the screenshot above.

Reply:
@chuckcarpenter @rwwagner90 This could be what we want, though. I was thinking the placement should move the arrow, but after revisiting both the Tippy and Popper demos, it appears that the main idea is to move the body of the tooltip.